package.json

```diff
@@ -32,10 +32,8 @@
     "es2015/text.js"
   ],
   "scripts": {
-    "build": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
-    "clean": "rm -rf -- index.js text.js es2015",
```

Comment:
Cool! I just changed it to be this…

```diff
diff --git a/package.json b/package.json
index 0437dd0..2cbdc38 100644
--- a/package.json
+++ b/package.json
@@ -32,7 +32,7 @@
     "es2015/text.js"
   ],
   "scripts": {
-    "build": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
+    "build": "rm -rf -- index.js text.js es2015; babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
     "test": "mocha",
     "test:watch": "npm run test -- --watch"
   },
```

but then realised the `babel` commands seem to nuke those files when it regenerates them, so there's no need to do it manually.

Unless I misunderstood? Happy to change it though! 😊